test/e2e/data/infrastructure-provider-byoh/v1beta1/templates/docker/clusterclass-quickstart-docker.yaml

```diff
@@ -2,37 +2,37 @@
 apiVersion: cluster.x-k8s.io/v1beta1
 kind: ClusterClass
 metadata:
-  name: quickstart
+  name: quickstart-docker
```

Comment:
NOTE: During release process, figured out clusterclass template file names should be different for VM and docker , that's why renamed `quickstart` to `quickstart-docker` for docker files.

cc: @shivi28

Reply:
Ya, it makes sense.

We can also merge it into a single yaml for better user experience, then we have to just release one yaml each for VM and docker to create a cluster using topology. I will raise a issue for this separately.

Reply:
Created a separate [issue](https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/issues/643) to merge files